renderdoc/driver/gl/wrappers/gl_draw_funcs.cpp

```diff
@@ -4538,21 +4538,24 @@ bool WrappedOpenGL::Serialise_glClear(SerialiserType &ser, GLbitfield mask)
             dstId = res_id;
           }
         }
+      }
 
+      if(dstId != ResourceId())
```

Comment:
Minor nitpick - this check is redundant as `GetOriginalID` will return if the id is null.